libs/indibase/defaultdevice.h

```diff
@@ -301,11 +301,6 @@ class DefaultDevice : public BaseDevice
          */
         virtual bool ISSnoopDevice(XMLEle *root);
 
-        /**
-         * @return getInterface Return the interface declared by the driver.
-         */
-        virtual uint16_t getDriverInterface() override;
```

Comment:
Why was this removed?

Reply:
#### Please look carefully:
Implementation in `DefaultDevice`:
https://github.com/indilib/indi/blob/e425d43bac5f716af11ecab10b2c2588d435f036/libs/indibase/defaultdevice.cpp#L796-L809
https://github.com/indilib/indi/blob/e425d43bac5f716af11ecab10b2c2588d435f036/libs/indibase/defaultdevice.cpp#L899-L904
Implementation in `BaseDevice`:
https://github.com/indilib/indi/blob/e425d43bac5f716af11ecab10b2c2588d435f036/libs/indibase/basedevice.cpp#L898-L908

#### Summary:
The implementations are equivalent.
For optimization, I suggest moving `interfaceDescriptor` to `BaseDevice`, and leaving property in `DefaultDevice` (so that there is still no property in `BaseDevice`)

That's right?

Reply:
We could since DefaultDevice inherits BaseDevice. However, BaseDevice was originally created as *Device-View* for Clients, so there was some separation in functions. We can consolidate now any duplicate features.